<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KT Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew_File_Ctrl_N"/>
    <addaction name="actionOpen_File_Ctrl_O"/>
    <addaction name="actionSave_Ctrl_s"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionClose_All"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
    </widget>
    <addaction name="actionSyntex"/>
    <addaction name="menuColors"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File_Ctrl_N">
   <property name="text">
    <string>New File (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File_Ctrl_O">
   <property name="text">
    <string>Open File...(Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Ctrl_s">
   <property name="text">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionSyntex">
   <property name="text">
    <string>Syntex</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>White</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
